bolt-simple-taproot.md

```diff
@@ -0,0 +1,994 @@
+# Extension BOLT XX: Simple Taproot Channels
+
+Authors:
+  * Olaoluwa Osuntokun <roasbeef@lightning.engineering>
+  * Eugene Siegel <eugene@lightning.engineering>
+
+Created: 2022-04-20
+
+# Table of Contents
+
+- [Introduction](#introduction)
+  * [Abstract](#abstract)
+  * [Motivation](#motivation)
+  * [Preliminaries](#preliminaries)
+    + [Pay-To-Taproot-Outputs](#pay-to-taproot-outputs)
+      - [Tapscript Tree Semantics](#tapscript-tree-semantics)
+      - [BIP 86](#bip-86)
+      - [Taproot Key Path Spends](#taproot-key-path-spends)
+      - [Taproot Script Path Spends](#taproot-script-path-spends)
+    + [MuSig2](#musig2)
+      - [Key Aggregation](#key-aggregation)
+      - [Nonce Generation](#nonce-generation)
+      - [Nonce Handling](#nonce-handling)
+      - [Signing](#signing)
+    + [Nothing Up My Sleeves Points](#nothing-up-my-sleeves-points)
+  * [Design Overview](#design-overview)
+  * [Specification](#specification)
+    + [Feature Bits](#feature-bits)
+    + [Channel Funding](#channel-funding)
+      - [`accept_channel` Extensions](#-accept-channel--extensions)
```

Comment:
this should probably be open channel extensions. Also, the link should be `#open_channel-extensions`